test_outputs/connlist/anp_banp_blog_demo_explain_output.txt

```diff
@@ -0,0 +1,94 @@
+
+##########################################
+# Specific connections and their reasons #
+##########################################
+----------------------------------------------------------------------------------------------------------------------------------------------------------------
+Connections between 0.0.0.0-255.255.255.255 => foo/myfoo[Pod]:
+
+Denied list:
+	Denied TCP, UDP, SCTP due to the following policies // rules:
+		Egress (Allowed) due to the system default (Allow all)
+		Ingress (Denied)
+			[NP] foo/allow-monitoring // Ingress (captured but not selected by any Ingress rule)
+
+----------------------------------------------------------------------------------------------------------------------------------------------------------------
+Connections between bar/mybar[Pod] => foo/myfoo[Pod]:
+
+Denied list:
+	Denied TCP, UDP, SCTP due to the following policies // rules:
+		Egress (Allowed) due to the system default (Allow all)
+		Ingress (Denied)
+			[NP] foo/allow-monitoring // Ingress (captured but not selected by any Ingress rule)
```

Comment:
rephrase explanation:
`captured but not selected by any Ingress rule`

to:
`foo/myfoo[Pod] is selected by the policy, but bar/mybar[Pod] is not selected by any Ingress rule`

Reply:
done